crates/store/src/db/sql.rs

```diff
@@ -1077,6 +1240,37 @@ pub(crate) fn schema_version(conn: &Connection) -> rusqlite::Result<usize> {
     conn.query_row("SELECT * FROM pragma_schema_version", [], |row| row.get(0))
 }
 
+/// Generates a simple insert SQL statement with values for the provided table, fields, and record
+/// number.
+fn insert_sql(table: &str, fields: &[&str], record_count: usize) -> String {
```

Comment:
nit: this file is pretty big already, I wonder if we should move out functions like this into something like `src/db/sql/utils.rs` (and rename the current `sql.rs` into `src/db/sql/mod.rs`). We could also move the `sql-queries` directory under the `src/db/sql/queries` directory.